ecl2df/common.py

```diff
@@ -651,43 +656,107 @@ def stack_on_colnames(
     return dframe
 
 
-def parse_zonemapfile(filename: str) -> Optional[dict]:
-    """Return a dictionary from (int) K layers in the simgrid to strings
+def is_color(input_string: str) -> bool:
+    """Checks if the input string is avalid color.
+    That is six-digit hexadecimal, three-digit hexadecimal or
+    given as a SVG color keyword name
+    """
+    if input_string.lower() in SVG_COLOR_NAMES:
```

Comment:
(would fail if someone puts in upper cases in the text file, but noone is supposed to do that..)

Reply:
Do you think I should do: SVG_COLOR_NAMES.lower() also?

Reply:
If we state that the code is case-insensitive, that is maybe smart.

Reply:
I have added it where the list is loaded in the beginning of the file